iocage_lib/ioc_json.py

```diff
@@ -2042,6 +2042,33 @@ def json_check_prop(self, key, value, conf):
                             _callback=self.callback,
                             silent=self.silent
                         )
+                elif key == 'interfaces':
+                    for iface in value.split(','):
+                        if ':' not in value:
+                            iocage_lib.ioc_common.logit(
+                                {
+                                    'level': 'EXCEPTION',
+                                    'message':
+                                    "Interfaces must be specified as a pair "
+                                    "or a triplet that includes a host interface "
+                                    "to be on the bridge.\n"
+                                    "EXAMPLE: vnet0:bridge0,vnet1:bridge1:em0"
+                                },
+                                _callback=self.callback,
+                                silent=self.silent
+                            )
+                        if iface.count(':') == 2 and iface.split(':')[2] not in netifaces.interfaces():
```

Comment:
@skarekrow I think this check should be done repeated on starting jail as well. It is possible that the interface no longer exists when a jail is started vs created. Thoughts ?

Reply:
@sonicaj Good point!

Reply:
Agreed! I'll throw that in and rebase.

Reply:
Thank you ;)